docs/src/howto-guides/027_remote_access_with_cumulocity.md

```diff
@@ -0,0 +1,41 @@
+# Cumulocity RemoteAcessConnect plugin
+
```

Comment:
Yes I would agree, a link to the Cumulocity IoT docs would be useful, as one of the default mistakes is to forget to add the `Remote access` admin permission to your user/role. Otherwise you don't see the `Remote access` tab in the Device Management application